### PR TITLE
Replace Apache copyright license with MIT

### PR DESCRIPTION
**Pull request by jcagme**
In Arcade we were using Apache and MIT licenses. Continue using just MIT's

### CODE REVIEW

File: src/Microsoft.DotNet.Github.IssueLabeler/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (eerhardt) [inline src/Microsoft.DotNet.Github.IssueLabeler/Properties/launchSettings.json]: This file seems unrelated to the current change.  Is it intended?

File: src/Microsoft.DotNet.Arcade.Sdk/Microsoft.DotNet.Arcade.Sdk.csproj [hunk 2]
```diff
@@ -1,4 +1,5 @@
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```
Comment (eerhardt) [inline src/Microsoft.DotNet.Arcade.Sdk/Microsoft.DotNet.Arcade.Sdk.csproj]: In the SignTool project we have

```
<!-- Copyright (c)  Microsoft.  All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

However, here we have a different license.

1. All the MSBuild files should have the same license.
2. Typically, other repos (corefx/coreclr) don't have licenses on the MSBuild files.  Do we need them in arcade?

Comment (eerhardt) [reply src/Microsoft.DotNet.Arcade.Sdk/Microsoft.DotNet.Arcade.Sdk.csproj]: Also, I think `dotnet/arcade` is a .NET Foundation project, right?  So we shouldn't have `Copyright (c) Microsoft`.  But of course, IANAL (I am not a lawyer).

Comment (jcagme) [reply src/Microsoft.DotNet.Arcade.Sdk/Microsoft.DotNet.Arcade.Sdk.csproj]: @markwilkie do you know or know who might know about these questions?

Comment (jcagme) [reply src/Microsoft.DotNet.Arcade.Sdk/Microsoft.DotNet.Arcade.Sdk.csproj]: @jaredpar told me that @richlander might have answers to these questions. @richlander do you know what's the best approach here?

**Comment by richlander** [comment]
I don't see a concern here. Just put the license header that we've adopted for msbuild files, whatever that is, including the 'null' header.

**Comment by jcagme** [comment]
Planning on merging this in the following ours. If we need to remove the headers from MSBuild files after talking to folks I will in following PRs
